ADAL/tests/ADAcquireTokenTests.m

```diff
@@ -780,6 +780,81 @@ - (void)testSilentExpiredATRefreshMRRTNetwork
     XCTAssertEqualObjects(mrrtItem.refreshToken, @"new refresh token");
 }
 
+- (void)testSilentExpiredATRefreshMRRTNetworkWithExpiresInUsingUnquotedNumber
```

Comment:
I agree, I'm not 100% sure how to make it better, this might be an improvement:

```
- (void)testAcquireTokenSilent_whenRedeemingMRRT_withNSNumbersInParsedJSON
```

I'm open to suggestions :)